get_letters.py

```diff
@@ -0,0 +1,19 @@
+#submitted by Keren Or Hadad 208025205
+
+def get_letters(): #o_ohel_p_python
+    """
+
+    :return: all the A-Z and a-z letters
```

Comment:
How? There is missing a documentation.
And describe what the variables do please.